From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: man-fignore

# DP: bash(1): mention quoting when assigning to FIGNORE

# DP: bash(1): mention quoting when assigning to FIGNORE
---
 doc/bash.1 | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/doc/bash.1 b/doc/bash.1
index 8177174..d17c288 100644
--- a/doc/bash.1
+++ b/doc/bash.1
@@ -2491,7 +2491,9 @@ A filename whose suffix matches one of the entries in
 .B FIGNORE
 is excluded from the list of matched filenames.
 A sample value is
-.Q .o:\*~ .
+.Q .o:\*~ 
+(Quoting is needed when assigning a value to this variable,
+which contains tildes).
 .TP
 .B FUNCNEST
 If set to a numeric value greater than 0, defines a maximum function
